lib/mobile_workflow.rb

```diff
@@ -1,4 +1,5 @@
 require "mobile_workflow/engine"
+require "mobile_workflow/displayable"
```

Comment:
What I did does not work well. When using the gem with a Rails app, it feels as if `Rails.application` has not been fully loaded when loading `displayable.rb`. Starting the Rails server triggers an error:
```
mobile_workflow/app/models/concerns/mobile_workflow/displayable.rb:5:in `<module:Displayable>': 
undefined method `routes' for nil:NilClass (NoMethodError)
```
which comes from this line of code `include Rails.application.routes.url_helpers` in displayable.rb#5. I am trying to fix it but it does not seem to work. Could you support me here, @FWMatt ? Thank you

Reply:
@FWMatt @MontseBadia I understand that the goal of this PR is to make `Displayable` not coupled/dependant on `Rails`. That is, instead of detecting if there is an attachment `attached?` (`ActiveStorage`) and generate attachment URLs from inside the class (`rails_representation_url ` and `rails_blob_url` from `Rails.application.routes.url_helpers`), do this task outside `Displayable`. But this will require to refactor all projects so they generate attachment URLs beforehand instead of delegating it to `Displayable`

Reply:
Seems to be working well now. Thanks @pjmartorell